owtf/webapp/package.json

```diff
@@ -3,11 +3,12 @@
   "version": "1.0.0",
   "description": "OWASP Offensive Web Testing Framework",
   "scripts": {
-    "build": "NODE_ENV=production webpack --config webpack/prod.config.js --color -p --progress --hide-modules --display-optimization-bailout",
+    "build": "NODE_ENV=production webpack --config webpack/prod.config.js --color  --progress ",
     "start": "NODE_ENV=development node server",
     "precommit": "pretty-quick --staged",
     "test": "jest",
-    "test:watch": "npm test -- --watch"
+    "test:watch": "npm test -- --watch",
```

Comment:
want to use yarn vs npm for consistency?

Reply:
Sure I will change it to yarn.